text/2020-05-14-compare-and-set.md

```diff
@@ -0,0 +1,20 @@
+# Compare-And-Set
+
+## Summary
+
+It allows to conditionally update a key, the condition being that the key has a certain value(maybe not exist).
+
+## Motivation
+
+It allows to implement read-modify-write scenarios.
+
+## Design
+
+- We can run the Compare-And-Set operation in the Scheduler which use Latch to control write concurrency.
```

Comment:
Can you add more details? I don't think people can get how it work by two sentences. 😆 

For example, what does the command look like? How does CAS interact with other commands? What changes need to be made for current implement? What does the whole data flow look like?

Reply:
Changed in https://github.com/tikv/rfcs/pull/51/commits/984955776458967d45bc3af9acdafbcc62617988. Please have another look.